core/src/rpc_subscriptions.rs

```diff
@@ -390,83 +412,97 @@ impl RpcSubscriptions {
     pub fn add_account_subscription(
         &self,
         pubkey: Pubkey,
-        confirmations: Option<Confirmations>,
+        commitment: Option<CommitmentConfig>,
         sub_id: SubscriptionId,
         subscriber: Subscriber<Response<RpcAccount>>,
     ) {
-        let mut subscriptions = self.account_subscriptions.write().unwrap();
+        let mut subscriptions = self.subscriptions.account_subscriptions.write().unwrap();
+        let last_notified_slot = if let Some((_account, slot)) = self
```

Comment:
This should be calculated using `commitment` right?

Reply:
Ah, yes. It wasn't necessary with the greater-than check in filter, but with the `!=`, this does need to be updated. Good catch!